dvc/command/version.py

```diff
@@ -37,7 +37,7 @@ def run(self):
             "Python version: {python_version}\n"
             "Platform: {platform_type}\n"
             "Binary: {binary}\n"
-            "Package manager: {package_manager}\n"
+            "Package type: {package_manager}\n"
```

Comment:
Let's refactor the names a bit:
```suggestion
            "Package: {package}\n"
```